interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -424,12 +425,25 @@ MathFuncts, errGen) {
             MathFuncts.mean(nodeObject.activityA) -
             MathFuncts.mean(nodeObject.activityB)
           );
-          nodeObject.logsig = -Math.log10(MathFuncts.tTest(
+          nodeObject.rawPValue = MathFuncts.tTest(
             nodeObject.activityA, nodeObject.activityB
-          ).pValue());
+          ).pValue();
 
           return nodeObject;
         });
+
+        // use FDR on the raw p-values from nodeInfoSet to get adjustedPValues
+        var rawPValues = nodeInfoSet.map(function getRawPValue(nodeObject) {
+          return nodeObject.rawPValue;
+        });
+        var adjustedPValues = MathFuncts.multTest.fdr(rawPValues);
+
+        // map the adjustedPValues back into the nodeInfoSet
+        nodeInfoSet = nodeInfoSet.map(function(nodeObject, i) {
```

Comment:
I have no idea why I would avoid `forEach()`, actually. I think you're right that it would be more efficient because we can modify the elements of the array in place rather than copying into a new array. Will fix.

- [ ] use `.forEach()` instead of `.map()` to modify array in place